Source/Renderer/RenderState.js

```diff
@@ -422,6 +422,23 @@ define([
         return cachedState;
     };
 
+    /**
+     * Use sparingly until we have a real solution for removing render states from the cache, e.g. TextureAtlas
+     * polutting the cache on resize.
+     *
+     * @private
+     */
+    RenderState.removeFromCache = function(renderState) {
```

Comment:
We need to reference count these, right?  What if someone else created the same render state?